src/_includes/templates/nav/primary-nav.html

```diff
@@ -2,7 +2,9 @@
 {% import 'macros/sub-nav.html' as sub_nav %}
 <nav class="primary-nav
             js-primary-nav"
-     aria-expanded="false">
+     aria-expanded="false"
+     aria-label="main navigation"
+     role="navigation">
```

Comment:
In most browsers `nav` has `role="navigation"` built in, but since [it doesn't in IE or Edge](html5accessibility.com), it makes sense to include it.